nilearn/datasets/data/MAIN_osf.csv

```diff
@@ -0,0 +1,84 @@
+participant_id,key_r,key_b
+sub-pixar001,5c8ff37da743a90018606df1,5c8ff37c2286e80019c3c102
+sub-pixar003,5c8ff37d4712b400193b5b54,5c8ff37d4712b400183b7011
+sub-pixar004,5c8ff37e2286e80016c3c2cb,5c8ff3832286e80019c3c10f
+sub-pixar005,5c8ff3822286e80018c3e37b,5c8ff382a743a90018606df8
+sub-pixar006,5c8ff3814712b4001a3b5561,5c8ff3832286e80016c3c2d1
+sub-pixar007,5c8ff3842286e80017c419e0,5c8ff3854712b4001a3b5568
+sub-pixar009,5c8ff3872286e80017c419ea,5c8ff3872286e80017c419e9
+sub-pixar010,5c8ff3884712b400183b7023,5c8ff3884712b400193b5b5c
+sub-pixar011,5c8ff389a743a9001660a016,5c8ff38c2286e80016c3c2da
+sub-pixar012,5c8ff38ca743a90018606dfe,5c8ff38ca743a9001760809e
+sub-pixar015,5c8ff38c4712b4001a3b5573,5c8ff38da743a900176080a2
+sub-pixar018,5c8ff38f2286e80018c3e38d,5c8ff3914712b4001a3b5579
+sub-pixar020,5c8ff391a743a900176080a9,5c8ff3914712b400173b5329
+sub-pixar022,5c8ff3912286e80018c3e393,5c8ff3952286e80017c41a1b
+sub-pixar027,5c8ff3952286e80016c3c2e7,5c8ff3954712b400193b5b79
+sub-pixar029,5c8ff395a743a900176080af,5c8ff3964712b400193b5b7d
+sub-pixar030,5c8ff399a743a9001660a031,5c8ff3982286e80017c41a29
+sub-pixar031,5c8ff39aa743a90018606e21,5c8ff39aa743a900176080ba
+sub-pixar035,5c8ff39aa743a900176080bf,5c8ff39d4712b400193b5b89
+sub-pixar039,5c8ff39ca743a90019606c50,5c8ff3a2a743a9001660a048
+sub-pixar041,5c8ff3a12286e80017c41a48,5c8ff3a12286e80016c3c2fc
+sub-pixar042,5c8ff39fa743a90018606e2f,5c8ff3a34712b4001a3b55a3
+sub-pixar044,5c8ff3a34712b400193b5b92,5c8ff3a84712b400183b7048
+sub-pixar046,5c8ff3a72286e80017c41a54,5c8ff3a7a743a90018606e42
+sub-pixar049,5c8ff3a74712b4001a3b55ad,5c8ff3a72286e80017c41a59
+sub-pixar050,5c8ff3aa4712b400183b704d,5c8ff3ac4712b4001a3b55b7
+sub-pixar052,5c8ff3aca743a9001660a063,5c8ff3ac4712b400183b7051
+sub-pixar057,5c8ff3ae4712b400183b7055,5c8ff3af2286e80018c3e3c0
+sub-pixar058,5c8ff3b02286e80018c3e3c4,5c8ff3b14712b400183b705a
+sub-pixar061,5c8ff3b12286e80016c3c30f,5c8ff3b34712b400183b7060
+sub-pixar062,5c8ff3b2a743a9001660a07a,5c8ff3b54712b400193b5ba3
+sub-pixar063,5c8ff3b2a743a9001660a07a,5c8ff3b54712b400193b5ba3
```

Comment:
These are now fixed ! See correct key :point_down:  
```suggestion
sub-pixar063,5c9e99d006cd47001a5ab599,5c8ff3b62286e80016c3c31b
```

Reply:
Will update this and see how CircleCI goes with it.

Thanks a lot!